test/tests/ver-1kb/ver-1kb.i

```diff
@@ -0,0 +1,211 @@
+    diffusivity = ${diffusivity}
+    block = '2'
+  []
+  [time_enclosure_2]
+    type = TimeDerivative
+    variable = concentration_enclosure_2
+    block = '2'
+  []
+[]
+
+[AuxKernels]
+  [pressure_enclosure_1]
+    type = ParsedAux
+    variable = pressure_enclosure_1
+    coupled_variables = 'concentration_enclosure_1'
+    expression = '${fparse R*T}*concentration_enclosure_1'
+    block = 1
+    execute_on = 'initial timestep_end'
+  []
+  [pressure_enclosure_2]
+    type = ParsedAux
+    variable = pressure_enclosure_2
+    coupled_variables = 'concentration_enclosure_2'
+    expression = '${fparse R*T}*concentration_enclosure_2'
+    block = 2
+    execute_on = 'initial timestep_end'
+  []
+[]
+
+[InterfaceKernels]
+  [interface_sorption]
+    type = InterfaceSorption
+    K0 = ${solubility}
+    Ea = 0
+    n_sorption = ${n_sorption}
+    diffusivity = ${diffusivity}
+    unit_scale = ${unit_scale}
+    unit_scale_neighbor = ${unit_scale_neighbor}
+    temperature = '${fparse T}'
+    variable = concentration_enclosure_1
+    neighbor_var = concentration_enclosure_2
+    sorption_penalty = 1e1
+    boundary = interface
+  []
+[]
+
+[Postprocessors]
+  [pressure_enclosure_1]
+    type = ElementAverageValue
+    variable = pressure_enclosure_1
+    block = 1
+    execute_on = 'initial timestep_end'
+  []
+  [pressure_enclosure_2]
+    type = ElementAverageValue
+    variable = pressure_enclosure_2
+    block = 2
+    execute_on = 'initial timestep_end'
+  []
+  [concentration_enclosure_1_at_interface]
+    type = PointValue
+    variable = concentration_enclosure_1
+    point = '${fparse 1/3 * long_total - 1e-5} 0 0'
+    outputs = 'csv console'
+    execute_on = 'initial timestep_end'
+  []
+  [pressure_enclosure_2_at_interface]
+    type = PointValue
+    variable = pressure_enclosure_2
+    point = '${fparse 1/3 * long_total + 1e-5} 0 0'
+    outputs = 'csv console'
+    execute_on = 'initial timestep_end'
+  []
+  [concentration_encl_1_inventory]
+    type = ElementIntegralVariablePostprocessor
+    variable = concentration_enclosure_1
+    block = 1
+    execute_on = 'initial timestep_end'
+  []
+  [concentration_encl_2_inventory]
+    type = ElementIntegralVariablePostprocessor
+    variable = concentration_enclosure_2
+    block = 2
+    execute_on = 'initial timestep_end'
+  []
+  [mass_conservation_sum_encl1_encl2]
+    type = LinearCombinationPostprocessor
+    pp_names = 'concentration_encl_1_inventory concentration_encl_2_inventory'
+    pp_coefs = '1            1'
+    execute_on = 'initial timestep_end'
+  []
+[]
+
+[Executioner]
+  type = Transient
+  end_time = ${simulation_time}
+  dtmax = 10
+  nl_abs_tol = 1e-14
+  nl_rel_tol = 1e-10
+  l_tol = 1e-3
+  scheme = 'bdf2'
+  solve_type = NEWTON
+  petsc_options_iname = '-pc_type -pc_factor_mat_solver_package'
+  petsc_options_value = 'lu superlu_dist'
+  [TimeStepper]
+    type = IterationAdaptiveDT
+    dt = 60
+    optimal_iterations = 12
```

Comment:
```suggestion
    dt = 1
    optimal_iterations = 4
```
a lot happens in the first time step, so it's OK to have it be small. The time stepper will increase it to larger value later on, anyway.